README.md

```diff
@@ -146,3 +146,7 @@ build using `UpdateDependencies.sh`:
 	cd chorus
 	./UpdateDependencies.sh
 
+##### Python
+
+Beginning with Ubuntu 20.04 (Focal), you will need to `sudo apt install python-is-python2` so that Mercurial can run.
```

Comment:
I'm not sure that's a wise thing to require since I imagine this will break a lot of other apps that are updated to run with Python 3. Wouldn't it be better to patch our Mercurial package to explicitly use python2 (`/usr/bin/python2` instead of `/usr/bin/python`)?